apis/fluent-en_GB/extensions/jdk8/atrium-api-fluent-en_GB-jdk8-jvm/src/main/kotlin/ch/tutteli/atrium/api/fluent/en_GB/jdk8/chronoLocalDateAssertions.kt

```diff
@@ -0,0 +1,19 @@
+package ch.tutteli.atrium.api.fluent.en_GB.jdk8
+
+import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.domain.builders.ExpectImpl
+import ch.tutteli.atrium.domain.builders.chronoLocalDate
+import java.time.chrono.ChronoLocalDate
+import java.time.chrono.ChronoLocalDateTime
+
+
+/**
+ * Expects that the subject of the assertion is before [expected] [ChronoLocalDate].
+ *
+ * @return This assertion container to support a fluent API.
+ * @throws AssertionError Might throw an [AssertionError] if the assertion made is not correct.
+ *
+ * @since 0.9.0
+ */
+fun <T : ChronoLocalDateTime<ChronoLocalDate>> Expect<T>.isBefore(expected: T): Expect<T> =
```

Comment:
There are two problems here:
- you mixed ChronoLocalDateTime up with ChronoLocalDate => use ChronoLocalDate here
- if you implement the same for ChronoLocalDateTime (in chronoLocalDateTimeAssertions), then use the signature as described in the description of the issue: `D: ChronoLocalDate, T : ChronoLocalDate<D>`